packages/component-library/src/CivicSandboxCard/CivicSandboxCard.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from 'emotion';
+import CivicSandboxCardDesktop from './CivicSandboxCardDesktop';
+import CivicSandboxCardMobile from './CivicSandboxCardMobile';
+
+const desktop = css`
+  display: block;
+
+  @media (max-width: 640px) {
+    display: none;
+  }
+`;
+
+const mobile = css`
+  display: none;
+
+  @media (max-width: 640px) {
+    display: block;
+  }
+`;
+
+const spacing = css`
+  margin: 80px auto;
+`;
+
+const CivicSandboxCard = ({
+  children,
+  mapLayers,
+  dashboardData,
+  title,
+  link,
+}) => (
+  <section className={spacing}>
+    <CivicSandboxCardDesktop
```

Comment:
One issue with rendering both components and setting to display none could be a performance hit on mobile browsers. the Desktop component will still get rendering updates. Not sure how big of a concern that is, but something to think about.

Reply:
Ooh, I don't like the sound of that, as I imagine that the desktop component is pretty intense, since it actually has a map in it. Do you have any other ideas?

Reply:
Note to self, use react-dimensions and a ternary to replace the css display: none.